packages/components/src/Inventory/WithHistory.tsx

```diff
@@ -0,0 +1,27 @@
+// @ts-nocheck
+import React, { useMemo } from 'react';
+import * as reactRouter from 'react-router-dom';
+import useChrome from '../useChrome';
+
+const WithReactRouterHistory = ({ Component, ...props }) => {
+  const history = reactRouter.useHistory();
```

Comment:
I'd like to point out that the history for router v6 no longer includes the basename. Meaning that modules using this HOC, will require to handle the `stable`/`preview` routing. Basename handling was moved to the router component level.